_data/master/navbars/reference.yml

```diff
@@ -85,6 +85,10 @@ toc:
     path: /reference/felix/configuration
   - title: Prometheus Statistics
     path: /reference/felix/prometheus
+- title: typha
```

Comment:
I feel like this should be capitalized: "Typha" ?

Reply:
OK, but then I think I should also capitalize the corresponding "felix" just above - is that OK?

Reply:
Yes, that would be great, thanks!